hackathon/README.md

```diff
@@ -7,23 +7,106 @@ MDAnalysis core developers on code --- it can be your own project
 where you want some help or start building something amazing with a
 few like-minded hackers!
 
-* date: **Friday 29 September, 2023**
-* time: 9:30 -- 15:00 (WEST)
-* location: [SANA Metropolitan
+**Discord server**: The [#hackathon][hackathon-channel] channel on the 
+[MDAnalysis Discord server][discord] is available for any questions, 
+sharing of ideas and discussion in the lead up to 
+the hackathon, during and after.
+
+* **Date**: Friday 29 September, 2023
+* **Time**: 10:00 -- 15:00 (WEST)
```

Comment:
```suggestion
* **Check-in**: 09:30 -- 10:00 (WEST)
* **Time**: 10:00 -- 15:00 (WEST)
```

Reply:
Just for skim readers who might see this and think "ah cool I'll show up at 10 on the dot"